changelogs/fragments/69-vmware-extract-repeat-code.yaml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+- vmware - Extracts repeated code from configure_vapp_properties() to set_vapp_properties() in vmware_guest.py.
```

Comment:
it's `vmware_guest`, not just `vmware`. Otherwise, everything looks good. I will approve as soon as this is fixed.

Reply:
Got it, fixed.

Reply:
thanks for your reactivity :+1:. I applied the `gate` label, it means the patchset is approved and our CI will rerun the test-suite and land the patch later today (ETA =~ 6h).

Reply:
Thank you for the help!

Reply:
You're welcome. Don't hesitate to push more good patches like this one :tada:.